### PR TITLE
Fifoplayer: Generate fake VI updates for fifologs which don't have them.

### PR DESCRIPTION
**Pull request by phire**
This is a prerequisite for #3880

---

This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/3974)

### CODE REVIEW

**Comment by lioncash** [comment]
Review status: 0 of 6 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[Source/Core/Core/FifoPlayer/FifoPlayer.cpp, line 28 \[r3\]](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3974#-KL_EqY1RR8-TK-UEx3_:-KL_EqY1RR8-TK-UEx3a:185253155) ([raw file](https://github.com/dolphin-emu/dolphin/blob/44661bba2d19932e2c9f3240791e5dbcce02b41e/Source/Core/Core/FifoPlayer/FifoPlayer.cpp#L28)):*
> ```cpp
> 
> bool IsPlayingBackFifologWithBrokenEFBCopies = false;
> bool IsPlayingBackFifologWithoutVIUpdates = false;
> ```

Since this is only ever queried externally, this should probably just be a getter function to keep it contained. Or alternatively, be made part of the FifoPlayer class itself with a non-static getter function.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3974)*

File: Source/Core/VideoCommon/VideoConfig.cpp [hunk 1]
```diff
@@ -108,6 +108,10 @@ void VideoConfig::Load(const std::string& ini_file)
   hacks->Get("EFBScaledCopy", &bCopyEFBScaled, true);
   hacks->Get("EFBEmulateFormatChanges", &bEFBEmulateFormatChanges, false);
 
+  // Force Virtual xfb on for testing:
```
Comment (degasus) [inline Source/Core/VideoCommon/VideoConfig.cpp]: these lines must not go to master, please drop them.

**Comment by degasus** [comment]
Else, looks good

**Comment by phire** [comment]
I'm currently re-writing this, wasn't happy with the communication between
videocommom and fifo player.

>

**Comment by leoetlino** [comment]
Looks like this is obsoleted by #5498 (5a372020).

**Pull request closed without merging**